src/sqlparse_fold.erl

```diff
@@ -502,15 +513,28 @@ fold(FType, Fun, Ctx, Lvl, {'create index', Opts, Idx, Table, Spec, Norm, Filter
               end,
     RT = {lists:append([
         "create ",
-        if length(OptsStr) > 0 -> OptsStr ++ " "; true -> [] end,
+        if length(OptsStr) > 0
+            -> OptsStr ++ " ";
+            true -> []
+        end,
         "index ",
-        if length(IdxStr) > 0 -> IdxStr ++ " "; true -> [] end,
+        if length(IdxStr) > 0
+            -> IdxStr ++ " ";
+            true -> []
+        end,
         "on ", TableStr,
         if length(Specs) > 0 ->
-            lists:append([" (", string:join(Specs, " | "), ")"]); true ->
-            "" end,
-        if NormStr =/= [] -> " " ++ NormStr; true -> [] end,
-        if FilterStr =/= [] -> " " ++ FilterStr; true -> [] end
+            lists:append([" (", string:join(Specs, " | "), ")"]);
+            true -> ""
+        end,
+        if NormStr =/= []
+            -> " " ++ NormStr;
+            true -> []
+        end,
+        if FilterStr =/= []
+            -> " " ++ FilterStr;
+            true -> []
+        end
```

Comment:
Please do not make any cosmetic change in the same PR

Reply:
Fixed - see above.